portfolio/src/main/java/com/google/sps/servlets/Comments.java

```diff
@@ -29,10 +29,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/comments")
+public class Comments extends HttpServlet {
```

Comment:
Maybe call it CommentServelet to be consistent

Reply:
Done: 29fdbd4